spam-prevention.md

```diff
@@ -65,15 +65,24 @@ network with a constant stream of such HTLCs to disrupt legitimate payments. We'
 
 ## Mitigation strategies available today
 
-It is not possible today to fully prevent this type of attacks, but properly configuring channels
-can help partially mitigate them:
+It is not possible today to fully prevent this type of attacks, but we can make the attacker's job
+harder by properly configuring channels:
+
+`htlc_minimum_msat * max_accepted_htlcs`
+
+As this equation is function of channel policy parameters, properly configuring them can help 
+partially mitigate them:
```

Comment:
I would remove these lines, and instead update the sentence below.

Reply:
Updated with sentence below.